src/hooks/oops.conf

```diff
@@ -1,13 +1,16 @@
-# Configuration file for Kernel oops hook
+# Configuration file for ABRT kernel oops hook
+# --------------------------------------------
 
-# If you want to see only reportable oopses,
-# set to "yes".
-# DropNotReportableOopses = yes
+# If enabled, only reportable oopses with useful backtraces will be preserved.
+#
+# Default: no
+#
+# DropNotReportableOopses = no
 
-# Lot of Machine Check Exceptions are correctable and thus not interesting to
-# users. Moreover some hardware may produce plenty of MCEs by design.
+# Many Machine Check Exceptions are corrigible and thus un interesting
```

Comment:
Call me stupid but I had to google what `corrigible` means. Why not keep the `correctable`? It is basically the same thing.

Reply:
"Correctable" sounds awful to me. I'll rephrase the sentence a little.